src/payment/unified.rs

```diff
@@ -5,29 +5,33 @@
 // http://opensource.org/licenses/MIT>, at your option. You may not use this file except in
 // accordance with one or both of these licenses.
 
-//! Holds a payment handler allowing to create [BIP 21] URIs with an on-chain, [BOLT 11], and [BOLT 12] payment
+//! Holds a payment handler allowing to create [BIP 21] URIs with on-chain, [BOLT 11], and [BOLT 12] payment
 //! options.
 //!
+//! Also allows to send payments using these URIs as well as [BIP 353] HRNs.
```

Comment:
This sentence has a double 'also'/'as well as'. Should it just say  'Also allows to pay to [BIP 353] human readable names'?